gordon_gcp/schemas/event.schema.json

```diff
@@ -1,3 +1,49 @@
 {
-    "foo": "bar"
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/event.schema.json",
+    "title": "Generic Event Messages",
+    "description": "Schema for parsing generic event messages",
+    "required": ["name", "rrdatas", "action", "type"],
+    "properties": {
```

Comment:
I also wonder if we should have a timestamp in this message as well.  I don't like the complexity it could add to Gordon, but given that we have only additions and deletions, an update to a record would need to be one deletion and one addition, in that order, and since order isn't necessarily guaranteed over pubsub, I can't think of a way besides timestamp to ensure it.

Reply:
yeah good call. It does look like messages pulled from pubsub will have a [publish_time](https://googlecloudplatform.github.io/google-cloud-python/latest/pubsub/subscriber/api/message.html) attribute (when using the python client library). But I also think it may be good for the publisher itself to add its own timestamp.